doc/multiple_robot_arms/multiple_robot_arms_tutorial.rst

```diff
@@ -0,0 +1,606 @@
+Introduction
+------------
+In MoveIt, we can plan motions for multiple robot arms, but there are pre-required steps to prepare robot models and configure ROS controllers. This tutorial provides ROS beginners with the steps to model multiple arms, configure controllers, and plan motions using MoveIt.
+
+.. image:: images/multiple_arms_start.png
+   :width: 600pt
+   :align: center
+
+Watch this `Quick YouTube video <https://www.youtube.com/watch?v=h8zlsuzeW3U>`_ of the multiple arms being controlled in Gazebo with the MoveIt Rviz plugin.
```

Comment:
FYI you don't need to use [Gazebo](https://github.com/PickNikRobotics/fake_joint); you could have used the fake_joint package to run a much simpler demo.

This is just a comment for your information, in case you did not know it already. No need to change it now.

Reply:
Got it. Thank you!